packages/auth/lib/password-policy/PasswordPolicyImpl.js

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2016-present Invertase Limited & Contributors
```

Comment:
Did you use any Google firebase-js-sdk code for this? If so I think we need to do the `Co-Authored-By: <foo@google.com>` for all authors trick that was used for vertexai, based on the committers for the file(s) you used